web3js/examples/ethGetBlock.ts

```diff
@@ -19,6 +19,8 @@ async function createWeb3Instance(options: LavaSDKOptions): Promise<Web3> {
   return new Web3(provider);
 }
 
+// backend usage with a private key
+// can be generated for free on gateway.lavanet.xyz
```

Comment:
What do you mean by `can be generated for free on gateway.lavanet.xyz`?
This is the private key option, what can be generated for free?